lib/tasks/webpacker.rake

```diff
@@ -2,6 +2,7 @@ tasks = {
   "webpacker:info"                    => "Provides information on Webpacker's environment",
   "webpacker:install"                 => "Installs and setup webpack with Yarn",
   "webpacker:compile"                 => "Compiles webpack bundles based on environment",
+  "webpacker:clean"                   => "Remove old compiled webpacks",
```

Comment:
Should probably be "Removes" to be consistent with other task descriptions

Reply:
Also would be great if this took a `keep` arg like the `assets:clean` rake task does : https://github.com/rails/sprockets/blob/master/lib/sprockets/manifest.rb#L245

Reply:
I'm mimicking `rails assets:clean` for my wording.

I've added the `keep` arg to the task. Good idea!

Reply:
I think the verbs of webpacker's tasks should follow singular verbs of `rails assets`. Send a separate PR?